modules/test/playwright/tests/client-extension-web/themeCSS.spec.ts

```diff
@@ -146,3 +146,28 @@ test('ThemeCSS client extension frontend token definition tokens appears stylebo
 		await clientExtensionsPage.deleteClientExtension(clientExtensionName);
 	});
 });
+
+test('LPD-37516 Sample control panel theme CSS client extension is properly deployed', async ({
+	clientExtensionsPage,
+	page,
+}) => {
+	const response = await page.goto(
+		'/o/liferay-sample-theme-css-4/css/main.css'
+	);
+
+	expect(response.status()).toBe(200);
+
+	await clientExtensionsPage.goto();
+
+	const background = await page
+		.locator('.control-menu')
+		.evaluate((controlMenu) => {
+			const computedStyle = window.getComputedStyle(controlMenu);
+
+			return computedStyle.background;
+		});
+
+	expect(background).toBe(
+		'rgba(0, 0, 0, 0) linear-gradient(105deg, rgb(0, 63, 91), rgb(43, 75, 125), rgb(95, 81, 149), rgb(152, 80, 157), rgb(204, 76, 145), rgb(242, 83, 117), rgb(255, 111, 78), rgb(255, 153, 19)) repeat scroll 0% 0% / auto padding-box border-box'
```

Comment:
The assert content differs from what is in the sample CSS file. That's because the browser translate what is there to what's here